@PestoOptions/PestoOptions.m

```diff
@@ -199,14 +199,7 @@
         % *     .GradObj ... are gradients provided?
         % *     .GradConstr ... do we have constraints?
         % *     .TolCon ... Tolerance for constraints
-        profileReoptimizationOptions = optimset( ...
-            'algorithm', 'interior-point', ...
-            'display', 'off', ...
-            'MaxIter', 400, ...
-            'GradObj', 'on', ...
-            'GradConstr', 'on', ...
-            'TolCon', 1e-6 ...
-            );
+        profileOptimizationOptions = [];
         
         % Maximal relative decrease of ratio allowed for two adjacent
         % points in the profile (default = 0.10) if options.dJ = 0;
```

Comment:
Changes here are due to Issue #131:
Now, Profile Optimization will use the same options as local optimization, unless specified differently. So getParameterProfiles checks for emptiness of this option.
Moreover, it's renamed to profileOptimizationOptions instead of profile"Re"optimizationOptions.